lxd/resources/cpu.go

```diff
@@ -41,42 +41,51 @@ func GetCPUIsolated() []int64 {
 	return isolatedCpusInt
 }
 
-// ParseCpuset parses a limits.cpu range into a list of CPU ids.
-func ParseCpuset(cpu string) ([]int64, error) {
-	cpus := []int64{}
-	chunks := strings.Split(cpu, ",")
+func parseRangedListToInt64Slice(input string) ([]int64, error) {
+	res := []int64{}
+	chunks := strings.Split(input, ",")
 	for _, chunk := range chunks {
 		if strings.Contains(chunk, "-") {
 			// Range
 			fields := strings.SplitN(chunk, "-", 2)
 			if len(fields) != 2 {
-				return nil, fmt.Errorf("Invalid cpuset value: %s", cpu)
+				return nil, fmt.Errorf("Invalid cpuset value: %s", input)
 			}
 
 			low, err := strconv.ParseInt(fields[0], 10, 64)
 			if err != nil {
-				return nil, fmt.Errorf("Invalid cpuset value: %s", cpu)
+				return nil, err
```

Comment:
I'm generally not a fan of seeing error contexts removed, what was the reason for this?